index.html

```diff
@@ -0,0 +1,146 @@
+            /></a></span>
+            <span class="close"
+              ><img
+                src="./images/phone-view-icons/x.svg"
+                alt="close-icon"
+                id="close-icon"
+            /></span>
+            <ul class="menu-list">
+              <li><a href="About.html" id="about">About</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Join</a></li>
+              <li><a href="#">Sponsor</a></li>
+              <li><a href="#" class="button" id="button">CC Campaign</a></li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <section class="Welcome-container">
+      <h1>Hello! Welcome To All.</h1>
+      <h2>THIERRY'S RESTAURANT 2.0</h2>
+      <p class="intro-para">
+        Join us in ST THOMAS LOGBABA, 2023 for an exciting launching of my
+        Restaurant, nice African food with various variety will be serve on that
+        day. The payement fee for the entrance is free! So what are you waiting
+        for come massively.
+      </p>
+      <p class="second-para">15 July 2023 FROM 02:00pm</p>
+      <p class="thierd-para">@ Cameroon , the City of Douala</p>
+    </section>
+    <section class="main-program-container">
+      <div class="program-container">
+        <div class="main-text">Main Program</div>
+        <div class="orange-line"></div>
+      </div>
+      <div class="program-details-container" id="program-details-container">
+      </div>
+      <div class="btn-mobile-program">SEE THE WHOLE PROGRAM</div>
+      <button type="button" class="btn-program">Join Us</button>
+    </section>
+    <section class="investors">
+      <h3>Featured Investors</h3>
+      <div class="orange-line"></div>
+      <div class="investors-container" id="container">
+      </div>
+    </section>
+    <section class="partner hidden">
+      <h3 class="partner-text">Partners</h3>
+      <div class="orange-line"></div>
+      <ul class="bottom-logos">
+        <li>
+          <a href="#"
+            ><img
+              class="bottom-logo"
+              src="./images/phone-view-icons/airbnb.png"
+              alt="logo"
+          /></a>
+        </li>
+        <li>
+          <a href="#"
+            ><img
+              class="bottom-logo"
+              src="./images/phone-view-icons/bloter.png"
+              alt="logo"
+          /></a>
+        </li>
+        <li>
+          <a href="#"
+            ><img
+              class="bottom-logo"
+              src="./images/phone-view-icons/naver.png"
+              alt="logo"
+          /></a>
+        </li>
+        <li>
+          <a href="#"
+            ><img
+              class="bottom-logo"
+              src="./images/phone-view-icons/kakao.png"
+              alt="logo"
+          /></a>
+        </li>
+      </ul>
+    </section>
+    <footer class="footer hidden">
+      <img
+        class="logo"
+        src="./images/phone-view-icons/main-footer.png"
+        alt="logo"
+      />
```

Comment:
- [ Optional ] I think it would be better if you customize this logo in the footer section of your design. According to the project requirements, all content should be customized. Customizing the content in your project gives the website a unique theme from other projects, I highly recommend you do this for the logo and any other content that is not customized.

## Your design
![Screenshot from 2023-07-06 23-02-52](https://github.com/axcell-0/first-capstone/assets/16392046/f2cdc8e8-ca59-4186-8458-a173475102f5)

## Project requirements
![Screenshot from 2023-07-06 23-09-26](https://github.com/axcell-0/first-capstone/assets/16392046/8f9c0a89-b28c-4e60-a105-2d8b7328926f)